transformation/src/main/scala/org/broadinstitute/monster/hca/HcaPipelineBuilder.scala

```diff
@@ -460,35 +440,17 @@ object HcaPipelineBuilder extends PipelineBuilder[Args] {
     )
   }
 
-  /**
-    * @param filenamesAndMsg The filenames and messages to validate
-    * @param inputPrefix The entire prefix before the filename
-    * @return the original SCollection
-    */
-  def validateJson(
-    filenamesAndMsg: SCollection[(String, Msg)],
-    inputPrefix: String
-  ): SCollection[(String, Msg)] = {
-    validateJsonInternal(inputPrefix)(filenamesAndMsg).withName("Validate: Log Validation").map {
-      case Some(error) =>
-        error.log
-      case None =>
-    }
-    filenamesAndMsg
-  }
-
   /**
     * @param inputPrefix The entire prefix before the filename
     * @param filenamesAndMsg The filenames and messages to validate
     * @return A collection of possible errors
     */
-  def validateJsonInternal(inputPrefix: String)(
+  def validateJson(inputPrefix: String)(
```

Comment:
I don't see why you need two arg lists for this function, does that affect the graph generation?

Reply:
The only reason was for running the function with input data in the tests; didn't have a good way of running it otherwise :/